released/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.1.0.2/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "coq-mathcomp-real-closed"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://github.com/math-comp/real-closed"
+bug-reports: "https://github.com/math-comp/real-closed/issues"
+dev-repo: "git+https://github.com/math-comp/real-closed.git"
+license: "CeCILL-B"
+
+build: [ make "-j" "%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq-mathcomp-field" { (>= "1.8.0" & < "1.9.0") }
+  "coq-mathcomp-bigenough"   {(>= "1.0.0" & < "1.1.0~")}
+]
+
+tags: [ "keyword:real closed field" "keyword:small scale reflection" "keyword:mathematical components" "date:2019-05-23" "logpath:mathcomp"]
```

Comment:
As per [CEP3](https://github.com/coq/ceps/blob/master/text/003-opam-metadata.md) a more focused `logpath` could be useful: `mathcomp.real_closed`. Also in the `extra-dev` package.